include/gridtools/meta/val.hpp

```diff
@@ -0,0 +1,44 @@
+/*
+ * GridTools
+ *
+ * Copyright (c) 2014-2021, ETH Zurich
+ * All rights reserved.
+ *
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#pragma once
+
+#if __cplusplus >= 201703
+
+#include "list.hpp"
+#include "macros.hpp"
+
+namespace gridtools::meta {
+
+    template <auto...>
+    struct val {};
```

Comment:
Can you add short description/example how and when this should be used.

Reply:
done